language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/CommandSubmissionClientImpl.java

```diff
@@ -24,18 +24,18 @@
 public class CommandSubmissionClientImpl implements CommandSubmissionClient {
 
   private final String ledgerId;
-  private final CommandSubmissionServiceGrpc.CommandSubmissionServiceFutureStub serviceStub;
+  private CommandSubmissionServiceGrpc.CommandSubmissionServiceFutureStub serviceStub;
```

Comment:
```suggestion
  private final CommandSubmissionServiceGrpc.CommandSubmissionServiceFutureStub serviceStub;
```
There's no actual need of making this `final`, forking a stub is cheap.